pkg/urlutil/urlutil_test.go

```diff
@@ -6,17 +6,21 @@ var (
 	gitUrls = []string{
 		"git://github.com/docker/docker",
 		"git@github.com:docker/docker.git",
-		"git@bitbucket.org:atlassianlabs/atlassian-docker.git",
```

Comment:
I just wanted a test without the `.git`, for a bit more diversity/coverage. If you think it shouldn't be changed, I'll change it back.